control/controlclient/netmap.go

```diff
@@ -39,6 +39,12 @@ type NetworkMap struct {
 	Hostinfo      tailcfg.Hostinfo
 	PacketFilter  []filter.Match
 
+	// ServicesCollectionAllowed is whether the node's Tailnet has
```

Comment:
Go has a standard phrase for documenting bools: "reports whether". (Sometimes "indicates whether", for fields.) So this would be:

```go
// CollectServices reports whether this node's Tailnet has requested that info about services be included in HostInfo.
```